apps/provisioning_api/lib/Controller/UsersController.php

```diff
@@ -130,15 +130,15 @@ public function __construct(
 	 *
 	 * 200: Users returned
 	 */
-	public function getUsers(string $search = '', ?int $limit = null, int $offset = 0): DataResponse {
+	public function getUsers(string $search = '', ?int $limit = null, int $offset = 0, string $orderBy = 'lastLogin', string $sort = 'DESC'): DataResponse {
```

Comment:
```suggestion
	public function getUsers(string $search = '', ?int $limit = null, int $offset = 0, string $orderBy = 'uid', string $sort = 'ASC'): DataResponse {
```
We should not break the existing sort order